packages/arb-token-bridge-ui/src/components/TransferPanel/TransferDisabledDialog.tsx

```diff
@@ -49,16 +78,56 @@ export function TransferDisabledDialog() {
       onClose={onClose}
     >
       <div className="flex flex-col space-y-4 py-4">
-        <p>
-          Unfortunately, <span className="font-medium">{unsupportedToken}</span>{' '}
-          has a custom bridge solution that is incompatible with the canonical
-          Arbitrum bridge.
-        </p>
-        <p>
-          For more information please contact{' '}
-          <span className="font-medium">{unsupportedToken}</span>
-          &apos;s developer team directly or explore their docs.
-        </p>
+        {/* teleport transfer disabled content if token is not in the allowlist */}
+        {isTeleportMode ? (
+          <>
+            <p>
+              Unfortunately,{' '}
+              <span className="font-medium">{unsupportedToken}</span> is not yet
+              supported for direct {sourceChainName} to {destinationChainName}{' '}
+              transfers.
+            </p>
+            {typeof l2ChainIdForTeleport !== 'undefined' && (
```

Comment:
`l2ChainIdForTeleport` is not used below, so I'd use `{ l2ChainIdForTeleportName && (` instead?